File: src/csharp/oop/MovieTheater/Cinemas/Booking/Order.cs
using MovieTheater.Cinemas.Layouts;
using MovieTheater.Cinemas.Screenings;

namespace MovieTheater.Cinemas.Booking;

public record Order(string Number, string UserId, Screening Screening)
{
    public List<(Seat, OrderSeatType)> Seats { get; } = [];
    public OrderStatus Status { get; set; } 
}

File: src/python/oop/MovieTheater/theater/order.py
from dataclasses import dataclass
from datetime import datetime

from .person_age import PersonAge
from .seat import Seat
from .screening import Screening

@dataclass
class Order:
    id: str
    user_id: str
    screening: Screening
    date: datetime
    seats: list[(Seat, PersonAge)]
